include/openssl/x509.h

```diff
@@ -683,8 +684,8 @@ int X509_certificate_type(const X509 *x, const EVP_PKEY *pubkey);
 
 long X509_REQ_get_version(const X509_REQ *req);
 int X509_REQ_set_version(X509_REQ *x, long version);
-X509_NAME *X509_REQ_get_subject_name(const X509_REQ *req);
-int X509_REQ_set_subject_name(X509_REQ *req, X509_NAME *name);
+const X509_NAME *X509_REQ_get_subject_name(const X509_REQ *req);
```

Comment:
Not for the return type

Reply:
See my respective question above

Reply:
Reverted as well for backward compatibility.